pr_agent/servers/bitbucket_server_webhook.py

```diff
@@ -50,15 +71,10 @@ async def root():
 
 
 def start():
-    bitbucket_server_url = get_settings().get("BITBUCKET_SERVER.URL", None)
-    if not bitbucket_server_url:
-        raise ValueError("BITBUCKET_SERVER.URL is not set")
-    get_settings().config.git_provider = "bitbucket_server"
-    middleware = [Middleware(RawContextMiddleware)]
-    app = FastAPI(middleware=middleware)
+    app = FastAPI(middleware=[Middleware(RawContextMiddleware)])
     app.include_router(router)
-    uvicorn.run(app, host="0.0.0.0", port=3000)
+    uvicorn.run(app, host="0.0.0.0", port=int(os.environ.get("PORT", "3000")))
```

Comment:
Please change this to BITBUCKET_SERVER.PORT, use get_settings().get(), add to configuration file

Reply:
It's like this because some cloud providers uses an environment variable PORT to bind the ingress... also in bitbucket_app.py is done like this:

https://github.com/Codium-ai/pr-agent/blob/main/pr_agent/servers/bitbucket_app.py#L162

Reply:
can't complain on my own code :(